client/blocks/express-checkout/styles.scss

```
@@ -0,0 +1,52 @@
+// on the <li> to prevent the invisible Stripe internals from inflating gaps.
+.wc-block-components-express-payment--cart {
+	.wc-block-components-express-payment__event-buttons {
+		display: flex !important;
+		flex-direction: column !important;
+		gap: 12px !important;
+
+		> li {
+			margin: 0 !important;
+			padding: 0 !important;
+			width: 100% !important;
+			// Collapse the inline-element line box so the iframe doesn't add
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)